ProjectLighthouse/Controllers/EnterLevelController.cs

```diff
@@ -62,6 +62,7 @@ public async Task<IActionResult> PlayLevel(int slotId)
             {
                 v = await visited.FirstOrDefaultAsync();
             }
+            if (v == null) return this.StatusCode(403, "");
 
             if (v == null)
             {
```

Comment:
Not sure why this is being checked and returned twice.

Reply:
ditto, https://github.com/LBPUnion/project-lighthouse/pull/62/commits/a7443c16a0e240b50b4026f98ad60d5e744fc6a5